.npmignore

```diff
@@ -1,2 +1,5 @@
-test
-.travis.yml
+/tsconfig.json
```

Comment:
I'm pretty sure this file shouldn't be published to npm, but I'd love confirmation.

Also, should `index.ts` be excluded given that `index.d.ts` is present?

Reply:
:+1:

You don't need to ship the .ts files either. I believe if you ship them by mistake, the compiler will still prefer the .d.ts files, but I'm not sure.

Reply:
Sounds good. I included `/index.ts` in this file, too.